src/bash.rs

```diff
@@ -135,13 +135,8 @@ pub fn write_completion_script<W: Write>(buffer: &mut W, command: &str, dfa: &DF
         write!(buffer, r#"
     if [[ -v "commands[$state]" ]]; then
         local command_id=${{commands[$state]}}
-        IFS=$'\n' read -r -d '' -a command_completions < <( _{command}_${{command_id}} "${{COMP_WORDS[$COMP_CWORD]}}" && printf '\0' )
-        for line in "${{command_completions[@]}}"; do
-            local elem=$(echo "$line" | cut -f1)
-            completions+=($elem)
-        done
+        completions+="$(_{command}_${{command_id}} "${{COMP_WORDS[$COMP_CWORD]}}" | cut -f1)"$'\n'
```

Comment:
Is the `cut -f1` necessary here? It strips everything after the first tab character.

Reply:
Hi, yes, it’s necessary.  The shell command within `_{command}_${{command_id}}` has to behave in the same way (i.e. produce both completions and descriptions) for all three shells since it’s the same command being used for all of them.  We just interpret that output in bash by stripping down the description, because Bash doesn’t support that.